src/js/components/Home/Countries.js

```diff
@@ -26,15 +48,16 @@ class Mapbuttons extends Component {
                     <div className="col-9">
                         <Map center={this.state.center}/>
                     </div>
+                    {/*<button onClick={()=>this.test("South Africa")}>press</button>*/}
```

Comment:
Add a TODO if you intend to use this line of code.

Reply:
Hi, just to let you know. I pushed this afternoon and worked now again and for some reason the work i'm busy with now also got included into the PR. Why I have no idea. I apologize for the incomplete code that made it way into the PR.

Reply:
Thanks for letting me know Marcel.

Reply:
@MrMarcelM Just to help you understand, when you have a PR open for the branch you are still on and you push new changes to that branch, it will automatically update the PR as well.